lib/matplotlib/backends/qt_editor/figureoptions.py

```diff
@@ -137,7 +137,7 @@ def prepare_data(d, init):
     for label in imagelabels:
         image = imagedict[label]
         cmap = image.get_cmap()
-        if cmap not in cm.cmap_d:
```

Comment:
I thought someone could accidentally create a custom cmap with the same name as one in cmap_d (although it's a pretty theoretical risk and it'd make the combobox awkward anyways), so checking on cmap identity should be safer.